CodePurpose/CodePurpose-Marketing-UI
# WIP adding firebase auth

## Describe Changes
I followed instructions from this tutorial to add react-router and firebase auth to the marketing app.
https://www.robinwieruch.de/complete-firebase-authentication-react-tutorial/
 ## How do I validate changes?
- Start the app, attempt to goto /admin
- validate you get kicket to an ugly login screen
- login with username `test@codepurpose.io` pass: `codepurpose88`
- validate after login you can successfully make it to /admin route and see placeholder
NOTE: This PR mostly reorganizes the code and adds the foundation for login and admin section but doesn't do much more. 
THINGS TO DO Next: Persist user in localstorage or context so that refresh doesn't remove logged in status, build out admin to have more functionality.
